code/socialDistribution/views.py

```diff
@@ -256,7 +256,7 @@ def un_befriend(request, author_id):
         curr_user = LocalAuthor.objects.get(user=request.user)
 
         if author.has_follower(curr_user):
-            author.followers.remove(curr_user)
+            author.follows.filter(actor=curr_user).delete()
```

Comment:
Would this delete the curr_user object or remove the reference in follows? 🤔

Reply:
It should remove the reference in follows I believe as we can follow and then unfollow and follow again.

Reply:
Ya, this would delete a Follow object (not an Author or LocalAuthor) so it should be good.